less/navigation.less

```diff
@@ -1 +1,19 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.top-page {
+    .center-align;
+    padding-bottom: 10px;
+    background: @nav-bg;
+    border-bottom: 2px solid @nav-border;
```

Comment:
Could also do a `position: absolute;` for a fixed nav. Good to try to implement some of that. Might have to adjust z-index as well, but all your code is clean.